### ISSUE TITLE
languagepackdownload always uses the default parameters

### ISSUE THREAD

**Issue by aronasorman** [learningequality/ka-lite#4058]
branch: 0.14.x

I'm trying to download pt-BR through the commandline using this command:

```
kalite manage languagepackdownload -l 'pt-BR' --commandline
```

However, the `lang_code` parameter always uses `settings.LANGUAGE_CODE`. Something's wrong with our parameter parsing.

**Comment by benjaoming** [comment]
I think the problem is that it isn't complaining about your "illegal" space after `-l`.

**Comment by benjaoming** [comment]
I think the problem is that it isn't complaining about your "illegal" space after `-l`.

**Comment by MCGallaspy** [comment]
"Fixes" in #4061... You should use the following command instead:
```
kalite manage languagepackdownload -l'pt-BR' --commandline
```

**Comment by aronasorman** [comment]
Fixed in the current 0.14.
